src/libraries/StableMath.sol

```diff
@@ -103,6 +104,13 @@ library StableMath {
                 return D;
             }
         }
+        /// @dev sometimes the iteration gets stuck in repeating loop
+        /// so if it is close enough we return it anyway
```

Comment:
Natspec is for interfaces, should use regular comment

Reply:
okay fixed